app/routes/git_tokens.py

```
@@ -95,75 +107,91 @@
     Retrieves all stored tokens with masked values for API response.
-    
+
     Returns:
         A list of dictionaries containing token metadata and masked token values.
     """
     try:
-        client = SupabaseClient()
-        res = client.select(table="git_label", columns="label, id, git_hosting,masked_token, created_at")
 
-        return res
+        query = f"SELECT label, id, git_hosting,masked_token, created_at FROM git_label   ORDER BY created_at DESC"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)